docs/topics/inline-classes.md

```diff
@@ -135,10 +153,26 @@ fun compute(x: UInt) { }
 To mitigate such issues, functions using inline classes are *mangled* by adding some stable hashcode to the function name. 
 Therefore, `fun compute(x: UInt)` will be represented as `public final void compute-<hashcode>(int x)`, which solves the clash problem.
 
-> `-` is an *invalid* symbol in Java, meaning that it's not possible to call functions which accept inline classes from Java.
+> Mangling schemes in Kotlin 1.4 and 1.4.30 are different. 
```

Comment:
What do you mean by 1.4? Can we specify one exact version when the scheme changed?